__tests__/app.test.js

```diff
@@ -182,6 +183,75 @@ describe("CORE: GET - /api/articles/:article_id", () => {
             expect(dates).toBeSorted({ descending: true })
           });
       });
+      test("200: Topic query within endpoint returns articles filtered by specific topic", () => {
+        return request(app)
+          .get("/api/articles?topic=mitch")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles.length).not.toBe(0)
+            body.articles.forEach((article) => {
+                expect(article).toHaveProperty("topic", ('mitch'))
+            })        
+          });
+      });
+      test("200: Invalid topic query within endpoint returns empty array", () => {
```

Comment:
Here you have described this as an invalid topic however if something is invalid would we expect at status code 200 🤔